parcelgen.py

```diff
@@ -121,6 +128,8 @@ def gen_parcelable_line(self, typ, member):
             return self.gen_array_parcelable(typ, memberized)
         elif self.list_type(typ):
             return self.gen_list_parcelable(typ, memberized)
+        elif self.map_type(typ):
+            return self.tabify("parcel.writeMap(%s);" % (memberized))
```

Comment:
Not sure it's a big deal, but docs recommend writeBundle() instead of writeMap(). Not sure if we care:

http://developer.android.com/reference/android/os/Parcel.html#writeMap(java.util.Map)